src/main/java/org/broadinstitute/consent/http/db/MailMessageDAO.java

```diff
@@ -20,8 +20,8 @@ public interface MailMessageDAO extends Transactional<MailMessageDAO> {
     Integer existsCollectDAREmail(@Bind("darReferenceId") String darReferenceId, @Bind("rpReferenceId") String rpReferenceId);
 
     @SqlUpdate("INSERT INTO email_entity " +
-            "(vote_id, entity_reference_id, user_id, email_type, date_sent, email_text) VALUES " +
-            "(:voteId, :entityReferenceId, :userId, :emailType, :dateSent, :emailText)")
+            "(vote_id, entity_reference_id, user_id, email_type, date_sent, email_text, create_date) VALUES " +
+            "(:voteId, :entityReferenceId, :userId, :emailType, :dateSent, :emailText, :dateSent)")
```

Comment:
perhaps I am misunderstanding, but when would `create_date` ever be anything different than `date_sent`? Should these values be separately inserted?

Reply:
`date_sent` could be null in the event that it was not sent.

Reply:
so instead of `create_date` being populated with `:dateSent`, it would be populated with a new argument to the DAO call, `:createDate`. So that it was possible for `create_date` and `date_sent` to have different values on insert.

Reply:
> so instead of `create_date` being populated with `:dateSent`, it would be populated with a new argument to the DAO call, `:createDate`. So that it was possible for `create_date` and `date_sent` to have different values on insert.

That's for a future story where we would distinguish between messages sent and not sent and therefore have different values. Today, all messages really are sent so they are, in fact, the same value. 

The reason we're populating the create date here is that we're adding that as a new column and we don't want to have a mix of entries that have create dates and some that do not. If we didn't put them here, current code paths would populate new entries with no creation date alongside back-populated ones that do have a creation date and we'd have a bit of a mess with inconsistent data.